frontend/.env.txt

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_BACKEND_PATH=http://127.0.0.1:8000
```

Comment:
Your shouldn't push env files in your commits.

Reply:
okk I'll remove my env and push again

Reply:
removed env, still if there's some changes i need to do pls do tell, its my first pr